tasks/start-template.html

```diff
@@ -43,6 +43,9 @@ <h2 class="cssdb-feature-heading">
 		<div class="cssdb-feature-content">
 			<p class="cssdb-feature-specification">
 				<a class="cssdb-feature-specification-link" href="${feature.specification}">Specification</a>
+				${feature.mdn
+					? `<a class="cssdb-feature-mdn-link" href="${feature.mdn}">MDN Docs</a>`
```

Comment:
If you create a new css class, you need add it into [`style.css`](https://github.com/csstools/cssdb/blob/gh-pages/style.css) from `gh-pages` branch.

Reply:
Yes, if we think it needs styling.

Reply:
And I agree with @nucliweb that we can add target blank to these.

```suggestion
					? `<a class="cssdb-feature-mdn-link" href="${feature.mdn}" target="_blank">MDN Docs</a>`
```